internal/provisioner/kops_provisioner_cluster_installation.go

```diff
@@ -193,6 +187,17 @@ func (provisioner *kopsCIAlpha) HibernateClusterInstallation(cluster *model.Clus
 	return nil
 }
 
+func configInstallationForHibernation(mattermost *mmv1alpha1.ClusterInstallation) {
+	// Hibernation is currently considered changing the Mattermost app deployment
+	// to 0 replicas in the pod. i.e. Scale down to no Mattermost apps running.
+	// The current way to do this is to set a negative replica count in the
+	// k8s custom resource. Custom ingress annotations are also used.
+	// TODO: enhance hibernation to include database and/or filestore.
+	mattermost.Spec.Replicas = hibernationReplicaCount
+	mattermost.Spec.IngressAnnotations = getHibernatingIngressAnnotations()
+	mattermost.Spec.Size = ""
```

Comment:
Is there a reason we are setting the size to `""` here? I don't think we want to do that necessarily. It has been a while since I reviewed the v1alpha1 operator logic, but I am pretty sure we shouldn't clear this value out.

Reply:
hi @gabrieljackson , Valid concern , Actually during local testing i found that operator overriding the replica=0 back to 1 based on size value which was not the expected behaviour for hibernated cluster installations. After checking with @Szymongib we decided to set the `size=""`.
And operator started respecting our hibernating values after setting the `size=""`. I had a concern about waking up same installation but testing shows that waking up also working as expected.  Does it answer your question ?

Reply:
Hmm, I reviewed the operator logic and see that there is some difference in handling between and empty size and not when it comes to individual resource settings. If your tests worked well with this then let's merge as is and we can investigate later if needed.